src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -22,4 +24,10 @@ export class GcsApiService extends GcsService {
     return this.http.get(url);
   }
 
+  public uploadFiles(bucketName: String, files: any[]) {
```

Comment:
If this is here, should we move `RegisterDownloadService.startDownload()` here as well? It's not clear to me why the methods are organized this way.